themes/theme-gmd/pages/Product/components/Media/index.jsx

```diff
@@ -1,48 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { isBeta } from '@shopgate/engage/core';
+import { isBeta, useWidgetSettings } from '@shopgate/engage/core';
 import { SurroundPortals } from '@shopgate/engage/components';
 import { PORTAL_PRODUCT_MEDIA_SECTION } from '@shopgate/engage/components/constants';
 import { ProductContext } from '@shopgate/engage/product';
+import ProductDiscountBadge from '@shopgate/engage/product/components/ProductDiscountBadge';
 import ProductImageSlider from './components/ProductImageSlider';
 import ProductMediaSlider from './components/ProductMediaSlider';
 
 /**
  * The product media component.
  * @returns {JSX}
  */
-const Media = ({ 'aria-hidden': ariaHidden, className }) => (
-  <ProductContext.Consumer>
-    {({ productId, variantId, characteristics }) => (
-      <SurroundPortals
-        portalName={PORTAL_PRODUCT_MEDIA_SECTION}
-        portalProps={{
-          productId,
-          variantId,
-        }}
-      >
-        {/* MediaSlider feature is currently in BETA testing.
+const Media = ({ 'aria-hidden': ariaHidden, className }) => {
+  const { pdp = { show: false, style: {} } } = useWidgetSettings('@shopgate/engage/product/components/ProductDiscountBadge');
```

Comment:
please move this inside the ProductDiscountBadge component, because Media component should not care about some configs for discount badge